### PR TITLE
Moved examples under eSignature, added private.key to .gitignore, add…

### PR DESCRIPTION
**Pull request by robkkni**
…ed quickstart to ds_config_sample, moved eg001_embedded_signing folder under app.

### CODE REVIEW

File: app/ds_config_sample.py [hunk 1]
```diff
@@ -24,7 +24,8 @@
     "gateway_name": "stripe",
     "gateway_display_name": "Stripe",
     "github_example_url": "https://github.com/docusign/code-examples-python/tree/master/app/",
-    "documentation": ""  # Use an empty string to indicate no documentation path.
+    "documentation": "",  # Use an empty string to indicate no documentation path.
+    "quickstart": "true"
```
Comment (InbarGazit) [inline app/ds_config_sample.py]: this should not be set to true here, it comes from the quickstart code. It should be :
  "quickstart" : '{QUICKSTART_VALUE}'

File: app/__init__.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
 # Register OAuth
 app.register_blueprint(ds)
 # Register examples
-app.register_blueprint(examples.eg001)
+app.register_blueprint(eg001_embedded_signing.eg001)
```
Comment (AaronWDS) [inline app/__init__.py]: change this back to examples

File: app/eg001_embedded_signing/controller.py [hunk 3]
```diff
@@ -5,9 +5,9 @@
     Recipients
 from flask import session, url_for, request
 
-from ...consts import authentication_method, demo_docs_path, pattern, signer_client_id
```
Comment (AaronWDS) [inline app/eg001_embedded_signing/controller.py:8]: change these back to three dots

File: app/eg001_embedded_signing/views.py [hunk 4]
```diff
@@ -6,9 +6,9 @@
 from flask import render_template, redirect, Blueprint
 
 from .controller import Eg001Controller
-from ...docusign import authenticate
-from ...ds_config import DS_CONFIG
```
Comment (AaronWDS) [inline app/eg001_embedded_signing/views.py:10]: change these back to 3 dots

**Review by AaronWDS — CHANGES_REQUESTED** [review]
We need to move the example 1 back into the folder with the other eSignature examples. Originally I was saying to update the [index template file](https://github.com/docusign/code-examples-python/blob/private/app/templates/home.html#L6) instead to check for the quickstart value and if the user is logged in. but at the moment I cannot confirm if you can send a redirect or run some arbitrary python code to get the redirect moving from within the template file. We should sync offline to discuss approaches

File: app/ds_config_sample.py (hunk 1 above)
Comment (robkkni) [reply app/ds_config_sample.py]: Oops! That was a mistake. Fixing!

File: app/__init__.py (hunk 2 above)
Comment (robkkni) [reply app/__init__.py]: Apparently I didn't understand how this works. Fixed. Thanks!

File: app/eg001_embedded_signing/controller.py (hunk 3 above)
Comment (robkkni) [reply app/eg001_embedded_signing/controller.py:8]: Leaving as-is to match CSharp launcher

File: app/eg001_embedded_signing/views.py (hunk 4 above)
Comment (robkkni) [reply app/eg001_embedded_signing/views.py:10]: Leaving as-is to match CSharp launcher

File: app/ds_config_sample.py (hunk 1 above)
Comment (robkkni) [reply app/ds_config_sample.py]: Fixed

**Review by AaronWDS — APPROVED** [review]
much better, lgtm 👍

**Pull request merged**